### PR TITLE
[13.0][ADD] stock_buffer_sales_analysis: sales analysis from stock buffer

### PR DESCRIPTION
**Pull request by carlosserratoro**

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]
LG

**Review by jgrandguillaume — APPROVED** [review]
LGTM thank you

**Review by TDu — APPROVED** [review]

File: stock_buffer_sales_analysis/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "Stock Buffer Sales Analysis",
+    "summary": "Allows to access the Sales Analysis from Stock Buffers",
+    "version": "13.0.1.0.0",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/ddmrp",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "category": "Warehouse",
+    "depends": ["ddmrp", "sale", "sales_team"],
+    "data": ["views/stock_buffer_views.xml"],
+    "installable": True,
```
Comment (guewen) [inline stock_buffer_sales_analysis/__manifest__.py:13]: Do we want auto_install here? I think we do

**Comment by LoisRForgeFlow** [comment]
@carlosserratoro It looks good, but as we recently proposed a "ddmrp_sale" module (https://github.com/OCA/ddmrp/pull/96)... Do you think it would be a good a idea to integrate this into ddmrp_sale?

**Review by jgrandguillaume — COMMENTED** [review]
LGTM

**Pull request merged**